doc/manuals/user/forbidden_characters.md

```diff
@@ -59,3 +59,42 @@ NGSIv2 introduces syntax restrictions for ID fields (such as entity id/type, att
 or metadata name/type) which are described in the "Field syntax restrictions" section in the
 [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/). You can also
 enable them for NGSIv1, as described in [this section of the documentation](v1_v2_coexistence.md#checking-id-fields).
+
+### Notification payload template
+
+NGSIv2 allows a templating mechanism (see "Notification templates" section in
+the [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/)). Forbidden
+characters restrictions apply to the `http.payload` field in NGSIv2 API operations, such as
+POST /v2/subscription or GET /v2/subscriptions.
+
+However, at notification time, any URL encoded characters is `http.payload` is decoded.
+
+Example:
+
+Let's consider the following `notification.http` object in a given subscription.
+
+```
+"http": {
+  "url": "http://foo.com/entity/${id}",
+  "headers": {
+    "Content-Type": "aplication/json"
+  },
+  "method": "PUT",
+  "qs": {
+    "type": "${type}"
+  },
+  "payload": "{ %22temperature%22: ${temperature}, %22asString%22: %22${temperature}%22 }"
```

Comment:
Perhaps we could write out the quotes

    "payload": "{ "temperature": ${temperature}, "asString": "${temperature}" }"

And under the example explain that the value of "payload" must be encoded?:

*Note that the value of the field "payload" must be URL-encoded, as it contains forbidden characters and otherwise will not be accepted*